apps/io-profile-async/src/main.ts

```diff
@@ -0,0 +1,16 @@
+import { CosmosClient } from "@azure/cosmos";
+import { getConfigOrThrow } from "./config";
+import { InfoFunction } from "./functions/info";
+
+const config = getConfigOrThrow();
+
+const cosmosClient = new CosmosClient({
+  endpoint: config.COSMOSDB_URI,
+  key: config.COSMOSDB_KEY
+});
+const database = cosmosClient.database(config.COSMOSDB_NAME);
```

Comment:
Are we querying Cosmos in this function? If not, we can remove all Cosmos references for now. 
Agree, @arcogabbo?